### DISCUSSION TITLE
Off topic, software implementation for universal eVscope style image intensifier?

### DISCUSSION THREAD

**Discussion by astrotater** [aaronwmorris/indi-allsky#1359] [General]
I’m working on a universal eVscope style image intensifier. It has a Raspberry Pi Zero 2W at its core, with a RPi HQ Camera and the “eyepiece” is a 1.28” TFT. It will have space for a 3,000 mAh lipo which should run all night. It will have two encoder knobs, one for gain and one for exposure. 
I’m hoping it can turn any go-to telescope with tracking into a basic eVscope. 

The final size will be roughly that of the “hand grenade” style large eyepieces on the market today. 

Basic functions will be a “live” view for target centering, and some sort of limited live stacking at the TFT eyepiece. EAA at the eyepiece instead of requiring all sorts of equipment. 
Any thoughts on how to implement this? I’m considering just a basic python script to interface with the camera, with the only user interfaces being the two rotary encoders. Maybe add more features in the future, but just the basics for now. 

![IMG_0247](https://github.com/aaronwmorris/indi-allsky/assets/171597037/64d91247-077f-4b71-a465-d8e14394b364)
![IMG_0246](https://github.com/aaronwmorris/indi-allsky/assets/171597037/abdd0128-87f0-4f31-81a9-c6b03ec7ba05)

**Comment by aaronwmorris** [comment]
What kind of lens are you planning for this device?  Allsky or a tighter view?

**Reply by Mooflotic** [reply]
Seems cool...what would you use as an image intensifier?
